### ISSUE TITLE
Incorrect CI description

### ISSUE THREAD

**Issue by sammo3182** [fsolt/dotwhisker#91]
The `alpha` for `style = "dotwhisker"` has been deprecated. Instead, users can specify `conf.level` to change the range of the whisker through `broom:tidy`. 

Need to correct the relevant descriptions in the vignette.

```r
dwplot(m1, alpha = 0.01) # not work

dwplot(m1, conf.level = .90)
```

Check line 249-250 of `dwplot.R`, though

**Comment by fsolt** [comment]
See also #93

**Comment by sammo3182** [comment]
Fixed by eaac2062668bcbf14bc2fab6cfc6fea445f94ffe; adding the `ci` option.
